.travis.yml

```diff
@@ -15,8 +15,8 @@ env:
   - JOB=test
 
 script:
-  - 'if [ "$JOB" = "test" ]; then make test-ci; fi'
-  - 'if [ "$JOB" = "test-coverage" ]; then make test-ci-coverage; fi'
+  - 'if [ "$JOB" = "test" ]; then CI=true make test-ci; fi'
```

Comment:
>By any chance, do you maybe an idea why the tests have to be run sequentially instead of parallel on Travis?

It's because most CI services don't handle parallel test runs well. You can read more about this in Jest's doc on [Troubleshooting CI](http://facebook.github.io/jest/docs/troubleshooting.html#tests-are-extremely-slow-on-docker-and-or-continuous-integration-server)